commands/georadius.md

```diff
@@ -58,11 +56,7 @@ Since `GEORADIUS` and `GEORADIUSBYMEMBER` have a `STORE` and `STOREDIST` option
 
 Breaking the compatibility with the past was considered but rejected, at least for Redis 4.0, so instead two read-only variants of the commands were added. They are exactly like the original commands but refuse the `STORE` and `STOREDIST` options. The two variants are called `GEORADIUS_RO` and `GEORADIUSBYMEMBER_RO`, and can safely be used in replicas.
 
-Both commands were introduced in Redis 3.2.10 and Redis 4.0.0 respectively.
-
-@history
-
-* `>= 6.2`: Added the `ANY` option for `COUNT`.
+Both commands were introduced in Redis 3.2.10 and Redis 4.0.0, respectively.
```

Comment:
shouldn't that be deleted (and make sure it's in the json file?

Reply:
Yep - the json is wrong, and this can be deleted.